dll/win32/shlwapi/shlwapi.rc

```diff
@@ -30,77 +30,77 @@
 #pragma code_page(65001)
 
 #ifdef LANGUAGE_DA_DK
-    #include "lang/shlwapi_Da.rc"
+    #include "lang/da-DK.rc"
 #endif
 #ifdef LANGUAGE_DE_DE
-    #include "lang/shlwapi_De.rc"
+    #include "lang/de-DE.rc"
 #endif
 #ifdef LANGUAGE_EN_US
-    #include "lang/shlwapi_En.rc"
+    #include "lang/en-US.rc"
 #endif
 #ifdef LANGUAGE_ES_ES
-    #include "lang/shlwapi_Es.rc"
+    #include "lang/es-ES.rc"
 #endif
 #ifdef LANGUAGE_FI_FI
-    #include "lang/shlwapi_Fi.rc"
+    #include "lang/fi-FI.rc"
 #endif
 #ifdef LANGUAGE_FR_FR
-    #include "lang/shlwapi_Fr.rc"
+    #include "lang/fr-FR.rc"
 #endif
 #ifdef LANGUAGE_HE_IL
-    #include "lang/shlwapi_He.rc"
+    #include "lang/he-IL.rc"
 #endif
 #ifdef LANGUAGE_HU_HU
-    #include "lang/shlwapi_Hu.rc"
+    #include "lang/hu-HU.rc"
 #endif
 #ifdef LANGUAGE_IT_IT
-    #include "lang/shlwapi_It.rc"
+    #include "lang/it-IT.rc"
 #endif
 #ifdef LANGUAGE_JA_JP
-    #include "lang/shlwapi_Ja.rc"
+    #include "lang/ja-JP.rc"
 #endif
 #ifdef LANGUAGE_KO_KR
-    #include "lang/shlwapi_Ko.rc"
+    #include "lang/ko-KR.rc"
 #endif
 #ifdef LANGUAGE_LT_LT
-    #include "lang/shlwapi_Lt.rc"
+    #include "lang/lt-LT.rc"
 #endif
 #ifdef LANGUAGE_NL_NL
-    #include "lang/shlwapi_Nl.rc"
+    #include "lang/nl-NL.rc"
 #endif
 #ifdef LANGUAGE_NO_NO
-    #include "lang/shlwapi_No.rc"
+    #include "lang/no-NO.rc"
 #endif
 #ifdef LANGUAGE_PL_PL
-    #include "lang/shlwapi_Pl.rc"
+    #include "lang/pl-PL.rc"
 #endif
 #ifdef LANGUAGE_PT_PT
-    #include "lang/shlwapi_Pt.rc"
+    #include "lang/pt-PT.rc"
 #endif
 #ifdef LANGUAGE_RO_RO
-    #include "lang/shlwapi_Ro.rc"
+    #include "lang/ro-RO.rc"
 #endif
 #ifdef LANGUAGE_RU_RU
-    #include "lang/shlwapi_Ru.rc"
-#endif
-#ifdef LANGUAGE_SK_SK
-    #include "lang/shlwapi_Sk.rc"
+    #include "lang/ru-RU.rc"
 #endif
 #ifdef LANGUAGE_SL_SI
-    #include "lang/shlwapi_Si.rc"
+    #include "lang/si-SI.rc"
```

Comment:
The file should be called `sl-SI` (es el, ES EYE)

Reply:
Good catch :-) Fixed.